app/src/main/java/com/duckduckgo/app/browser/BrowserTabFragment.kt

```diff
@@ -830,6 +843,20 @@ class BrowserTabFragment : Fragment(), FindListener {
         }
     }
 
+    @SuppressLint("NewApi")
+    private fun launchAddWidget() {
+        val theme = (activity as? DuckDuckGoActivity)?.appTheme()
+        val widgetClass = if (theme == LIGHT) SearchWidgetLight::class.java else SearchWidget::class.java
+        val provider = ComponentName(context, widgetClass)
+        AppWidgetManager.getInstance(context).requestPinAppWidget(provider, null, null)
+    }
+
+    private fun launchLegacyAddWidget() {
+        val context = context ?: return
+        val options = ActivityOptions.makeSceneTransitionAnimation(this.activity, ctaOkButton, "animation")
```

Comment:
where is the transition name `"animation"` used?

Reply:
That variable is a view name not the transition name.

Reply:
Is the intention that the "ctaOkButton" will have a shared element transition to the Lottie animation in the instruction screen? if so, that's not happening for me.

Reply:
This was a bit confused, so I have deleted it entirely and used a straightforward activity to activity scene transition.